lips/lip-1.mdx

```diff
@@ -542,3 +539,9 @@ THIS API IS PROVIDED "AS IS" WITH NO EXPRESS OR IMPLIED WARRANTIES WHATSOEVER, I
 The Libra Association disclaims all liability relating to this API and to the
 implementation of this API, and disclaims all liability for cost of procurement of substitute services, lost profits, loss of use, loss of data or any incidental, consequential, direct,
 indirect, or special damages, whether under contract, tort, warranty or otherwise, arising in any way out of use or reliance upon this API or any information herein.
+
+The compliance processes described in this LIP are for informational purposes only and do not reflect the specific compliance obligations of VASPs under applicable regulatory frameworks, their compliance programs, and/or standards imposed by Libra Networks.
+ 
+# Copyright Notice
```

Comment:
Interesting to have a copyright notice at the end of a document like this. I would have thought the `LICENSE-DOCS` license file in the `lip` GitHub repo would have covered this. No real objection from me here if legal said to add this, but it did look odd to me at first glance.

Reply:
I just do what legal tells me :D